src/main/java/com/obj/nc/flows/dataSources/properties/http/HttpDataSourceProperties.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2021 the original author or authors.
+ * This file is part of Notiflow
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program. If not, see https://www.gnu.org/licenses/.
+ */
+
+package com.obj.nc.flows.dataSources.properties.http;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.validation.constraints.NotEmpty;
+
+@Getter
+@Setter
+public class HttpDataSourceProperties {
+    @NotEmpty
+    private String name;
+    @NotEmpty
+    private String url;
+
+    private String token;
+
+    private String pojoFCCN;
+
+    private String spelFilterExpression;
+
+    @NotEmpty
+    private String cron;
+
+    private String externalIdKey;
```

Comment:
Is this name of the name of the attribute in JSON that is considered to be PK? If so let's call it externalIDAttributeName

Reply:
renamed